environment_python_3_5.yml

```diff
@@ -25,7 +24,6 @@ dependencies:
 - setuptools=41.0.1
```

Comment:
looks like this is using python 3.6.4 instead of python 3.5.6?

Reply:
@jbcurtin How can you tell this?

Reply:
Expanding the code-snippet doesn't allow me to comment on the exact line, here is the line number from the whole file
https://github.com/spacetelescope/jwql/blob/0a987316f7f94837ed47c1e33dde8e46af3875e3/environment_python_3_5.yml#L20

Reply:
Ah yes, good catch!